docs/aws-iam-setup.md

```diff
@@ -0,0 +1,120 @@
+# Setting up AWS IAM authentication
+
+This document provides guidelines how to configure Percona Server for MongoDB to use AWS IAM authentication. The use of this authentication method enables you to natively integrate Percona Server for MongoDB with AWS services, increase security of your infrastructure by setting up password-less authentication and offload your DBAs from managing different sets of secrets. To learn more, see [AWS IAM authentication](aws-iam.md)
+
+To configure AWS IAM authentication means to set up your AWS environment and configure Percona Server for MongoDB. The AWS environment setup is out of scope of this document. Consult the AWS documentation to perform the following setup steps: 
+
+1.	[Configure the AWS service to work with IAM and support temporary credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_aws-services-that-work-with-iam.html)
+2.	For user authentication: 
+      
+      * [Create the IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) and copy its ARN (Amazon Resource Name)
+
+    For role authentication:
+
+      * [Create the IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html)
+      * Attach the IAM role to the AWS service.
+      * Copy the ARN of the IAM role. 
+
+## Configure Percona Server for MongoDB
+
+The steps are the following:
+
+1. Add records for the IAM user/role in the `$external` database
```

Comment:
Probably more clear wording would be "Create user records..." instead of "Add records..."

Reply:
Updated